recipes/conda-wrappers/meta.yaml

```diff
@@ -0,0 +1,28 @@
+{% set version = "1.0.0" %}
+
+package:
+  name: conda-wrappers
+  version: {{ version }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+  run:
+    - python
+    - exec-wrappers 1*
```

Comment:
> Yep, works great. We do it with lots of things.

Good to know it is working, I prefer using the dot. But the semantic of ```1*``` and ```1.*``` is the same, right?